src/Whoops/Exception/Inspector.php

```diff
@@ -92,45 +112,45 @@ public function getPreviousExceptionInspector()
     public function getFrames()
     {
         if ($this->frames === null) {
-            $frames = $this->getTrace($this->exception);            
-            
-            // Fill empty line/file info for call_user_func_array usages (PHP Bug #44428) 
+            $frames = $this->getTrace($this->exception);
```

Comment:
fixed in https://github.com/filp/whoops/pull/502/files .. will rebase after #502 landed